releasenotes/notes/commutation-checker-utf8-47b13b78a40af196.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features_transpiler:
+  - |
+    Added support for utf-8 quantum gate names in :class:`.CommutationChecker`.
```

Comment:
```suggestion
fixes:
  - |
    Fixed an issue with the :class:`.CommutationChecker` class where it would error if a gate's
    :attr:`~.Gate.name` attribute was UTF8 encoded. Previously only gate names with ascii
    encoding would work.
    Fixed `#12501 <https://github.com/Qiskit/qiskit/issues/12051>`__
```